cybershell/plugin_loader.py

```
@@ -0,0 +1,23 @@
+import importlib.util, sys
+from pathlib import Path
+from typing import Dict, Type
+from .plugins import PluginBase
+
+def load_user_plugins(dir_path: str) -> Dict[str, Type[PluginBase]]:
+    registry = {}
+    base = Path(dir_path)
+    if not base.exists(): return registry
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)